tests/validators/answers/test_number_answer_validator.py

```diff
@@ -80,6 +80,107 @@ def test_are_decimal_places_valid():
     }
 
 
+def test_maximum_set_as_string():
```

Comment:
Because these 3 tests are identical except for the minimum and maximum, you could parameterize them 
```
@pytest.mark.parameterize("minimum, maximum", [({"value": "100"}, None), (None, {"value": "0"}), ({"value": "100", {"value": "0"})])
```
and cut the repeated code down